packages/snaps-utils/src/checksum.test.ts

```diff
@@ -13,48 +15,74 @@ describe('checksum', () => {
   // echo -n 'foo bar' | shasum -a 256 | cut -d ' ' -f 1| xxd -r -p | base64
   const FOO_BAR_SHA256 = '+8Gp+Fjqnhd5FpZL2Iw9N7kaHoRBJ2XimVB3fyZcS3U=';
 
-  it('takes string', () => {
-    expect(base64.encode(checksum(FOO_BAR_STR))).toBe(FOO_BAR_SHA256);
+  it('takes string', async () => {
+    expect(base64.encode(await checksum(FOO_BAR_STR))).toBe(FOO_BAR_SHA256);
   });
 
-  it('takes Uint8Array', () => {
-    expect(base64.encode(checksum(FOO_BAR_UINT8))).toBe(FOO_BAR_SHA256);
+  it('takes Uint8Array', async () => {
+    expect(base64.encode(await checksum(FOO_BAR_UINT8))).toBe(FOO_BAR_SHA256);
   });
 
-  it('takes VirtualFile', () => {
-    expect(base64.encode(checksum(new VirtualFile(FOO_BAR_STR)))).toBe(
+  it('takes VirtualFile', async () => {
+    expect(base64.encode(await checksum(new VirtualFile(FOO_BAR_STR)))).toBe(
       FOO_BAR_SHA256,
     );
 
-    expect(base64.encode(checksum(new VirtualFile(FOO_BAR_UINT8)))).toBe(
+    expect(base64.encode(await checksum(new VirtualFile(FOO_BAR_UINT8)))).toBe(
       FOO_BAR_SHA256,
     );
   });
 
-  it('works on empty string', () => {
-    expect(base64.encode(checksum(''))).toBe(EMPTY_SHA256);
-    expect(base64.encode(checksum(new Uint8Array()))).toBe(EMPTY_SHA256);
+  it('works on empty string', async () => {
+    expect(base64.encode(await checksum(''))).toBe(EMPTY_SHA256);
+    expect(base64.encode(await checksum(new Uint8Array()))).toBe(EMPTY_SHA256);
+  });
+
+  it('uses crypto.subtle when it is available', async () => {
+    const crypto = webcrypto;
```

Comment:
This seems unneccesary?

Reply:
Yeah, I refactored this and forgot to clean up. Will do